<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressSetDialog</class>
 <widget class="QDialog" name="AddressSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frameAddressSetting">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>580</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_other_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>73</y>
     <width>280</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px</string>
   </property>
   <property name="text">
    <string>门铃类型</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_other_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>143</y>
     <width>280</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px</string>
   </property>
   <property name="text">
    <string>可视门铃开锁时限 (秒)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditDoorTimeout">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>143</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;
font:24px</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>10</y>
     <width>510</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>其他</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonDoorBellType">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>180</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>非可视门铃</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommonPushButton</class>
   <extends>QPushButton</extends>
   <header>commonpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
